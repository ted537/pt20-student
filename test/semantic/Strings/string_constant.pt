using output;
val x = "Hello";
