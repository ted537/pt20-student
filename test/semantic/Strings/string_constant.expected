   .tFileDescriptor
   .tLiteralInteger
   oEmitValue
   % value emitted 2
   .tFileBind
   .tLiteralAddress
   oEmitDataAddress
   % value emitted 0
   .tStoreInteger
     .tSkipString
     oEmitNullAddress
     % value emitted -32767
     .tStringData
     oEmitValue
     % value emitted 5
     oEmitString
     % value emitted 72
     % value emitted 101
     % value emitted 108
     % value emitted 108
     % value emitted 111
     % value emitted 0
    .tLiteralAddress
    .tFetchChar
    oEmitValue
    % value emitted 15
 .tTrapBegin
 .tTrap
 oEmitTrapKind(trHalt)
 % value emitted 0
